jetbrains-core/src/software/aws/toolkits/jetbrains/core/AwsClientManager.kt

```diff
@@ -32,8 +33,8 @@ open class AwsClientManager(project: Project, sdkClient: AwsSdkClient) :
 
         val busConnection = ApplicationManager.getApplication().messageBus.connect(project)
         busConnection.subscribe(CredentialManager.CREDENTIALS_CHANGED, object : ToolkitCredentialsChangeListener {
-            override fun providerRemoved(providerId: String) {
-                invalidateSdks(providerId)
+            override fun providerRemoved(identifier: ToolkitCredentialsIdentifier) {
+                invalidateSdks(identifier.id)
```

Comment:
We can't since the hash map is populated by the Provider, not Identifier. We could make the provider expose its corresponding Identifier though, but I would want to keep the .id and .displayName for code compactness.... Thoughts?